src/DynamicData/GlobalConfig.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Reactive.Concurrency;
+
+namespace DynamicData;
+
+internal static class GlobalConfig
+{
+    public static IScheduler DefaultScheduler { get; } = TaskPoolScheduler.Default;
```

Comment:
@RolandPheasant Do you think this should be a public thing so that consumers can change the value if they want to?

It does sound like a foot-gun, but maybe they have a reason for doing everything on the Intermediate Scheduler...  Who can say?